css/style.css

```diff
@@ -0,0 +1,327 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.container {
+  margin: 10px;
+  padding: 10px;
+  width: 100%;
+  height: 100%;
+}
+
+#title-text {
+  color: white;
+}
```

Comment:
[OPTIONAL] Using color names is considered bad practice. I suggest you change this to its `rgb` or `hex` equivalent. This comment applies to all of your colors, I'm marking just this one for brevity.

Reply:
done as advised